app/controllers/api/alerts_statuses_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Api
+  class AlertsStatusesController < BaseController
+    include Subcollections::AlertStatusStates
+
+    def update
+      if @req.action == "providers_alerts"
+        providers_alerts_resource
+      else
+        super
+      end
+    end
+
+    def providers_alerts_resource
+      payload = {"providers" => []}
+      ExtManagementSystem.all.each do |provider|
+        payload["providers"] << {
+          "environment" => "production",
+          "name"        => provider.name,
+          "type"        => provider.class.to_s,
+          "id"          => provider.id}.merge("alerts" => alerts_and_states(provider))
+      end
+      render_resource :alertes_statuses, payload
+    end
+
+    private
+
+    def alerts_and_states(provider)
+      result = []
+      provider.miq_alert_statuses.each do |alert_status|
+        alert = {
+          "evaluated_on"  => alert_status.evaluated_on,
```

Comment:
The id should also be included as it will be necessary in order to add/update states.